notebooks/py/assay_comparison.py

```diff
@@ -0,0 +1,237 @@
+spots(merfish, background_image=merfish_dots, spots_kwargs=dict(alpha=1.), ax=ax3)
+spots(merfish, background_image=merfish_nuclei, spots_kwargs=dict(alpha=1.), ax=ax5)
+spots(decoded_image=merfish_decoded_image, decoded_image_kwargs=dict(alpha=1.), ax=ax2)
+spots(decoded_image=merfish_decoded_image, background_image=merfish_dots, decoded_image_kwargs=dict(alpha=1.), ax=ax4)
+spots(decoded_image=merfish_decoded_image, background_image=merfish_nuclei, decoded_image_kwargs=dict(alpha=1.), ax=ax6);
+# EPY: END code
+
+# EPY: START markdown
+#From these examples, we can see that the point cloud over-estimates the spot size (perhaps we're calculating radius wrong?)
+# EPY: END markdown
+
+# EPY: START markdown
+### Show different Dartfish Spots Plots
+# EPY: END markdown
+
+# EPY: START code
+f, ((ax1, ax2), (ax3, ax4), (ax5, ax6)) = plt.subplots(ncols=2, nrows=3, figsize=(30, 45))
+spots(dartfish, background_image=np.zeros_like(dartfish_dots), spots_kwargs=dict(alpha=1.), ax=ax1)
+spots(dartfish, background_image=dartfish_dots, spots_kwargs=dict(alpha=1.), ax=ax3)
+spots(dartfish, background_image=dartfish_nuclei, spots_kwargs=dict(alpha=1.), ax=ax5)
+spots(decoded_image=dartfish_decoded_image, decoded_image_kwargs=dict(alpha=1.), ax=ax2)
+spots(decoded_image=dartfish_decoded_image, background_image=dartfish_dots, decoded_image_kwargs=dict(alpha=1.), ax=ax4)
+spots(decoded_image=dartfish_decoded_image, background_image=dartfish_nuclei, decoded_image_kwargs=dict(alpha=1.), ax=ax6);
+# EPY: END code
+
+# EPY: START markdown
+### Show different background types for ISS
+# EPY: END markdown
+
+# EPY: START code
+f, ((ax1, ax2), (ax3, ax4), (ax5, ax6)) = plt.subplots(ncols=2, nrows=3, figsize=(30, 40))
+spots(iss, background_image=np.zeros_like(iss_dots), spots_kwargs=dict(alpha=1.), ax=ax1)
+spots(iss, background_image=iss_dots, spots_kwargs=dict(alpha=1.), ax=ax3)
+spots(iss, background_image=iss_nuclei, spots_kwargs=dict(alpha=1.), ax=ax5)
+
+# ISS doesn't have a decoded image, so leave those plots blank to make this clear
+for ax in (ax2, ax4, ax6):
+    ax.set_axis_off()
+f.tight_layout()
+# EPY: END code
+
+# EPY: START markdown
+### Download available copy number information from original authors
+# EPY: END markdown
+
+# EPY: START code
+dartfish_copy_number = pd.read_csv('https://dmf0bdeheu4zf.cloudfront.net/20180821/DARTFISH/fov_001/counts.csv', index_col=0, squeeze=True)
+merfish_copy_number = pd.read_csv(data_root + "merfish_copy_number_benchmark.csv", index_col=0, squeeze=True)
+iss_copy_number = pd.read_csv(data_root + "iss_copy_number_benchmark.csv", index_col=1)['cnt']
+# EPY: END code
+
+# EPY: START code
+f, axes = plt.subplots(ncols=3, nrows=4, figsize=(12, 14))
+
+for dataset, benchmark, axis in zip([iss, merfish, dartfish], [iss_copy_number, merfish_copy_number, dartfish_copy_number], axes[0, :]):
+    compare_copy_number(dataset, benchmark, ax=axis, color='tab:blue')
+
+for dataset, axis in zip(datasets, axes[1, :]):
```

Comment:
these three plots (magnitude, area, distance) should have a parameter for 'threshold' which indicates the threshold that was picked as input to the SpotDetectors. The plot should show a prominent vertical line at that threshold on top of the histogram.